hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNode.java

```diff
@@ -1444,5 +1449,19 @@ public BRTrackingService getBRTrackingService(){
   NameNodeRpcServer getNameNodeRpcServer(){
     return rpcServer;
   }
+
+  static void createLeaseLocks(Configuration conf) throws IOException {
```

Comment:
Is there any way this can be updated without formating hdfs?

Reply:
yes see line 626

Reply:
perfect.